app/views/testing/done.html

```diff
@@ -42,11 +42,13 @@ <h2>What you can do next</h2>
       </ul>
 
       <h2>Tell others about this service</h2>
-      <ul>
-        <li><a href="https://twitter.com/intent/tweet?text=I%20just%20told%20the%20NHS%20about%20my%20experience%20of%20coronavirus&#46;%20If%20you%20haven&#39;t%20already&#44;%20you%20should%20too&#33;%20https%3A%2F%2Fnhs.uk/coronavirus-status-checker%2F" target="_blank">Share on Twitter</a> (opens in a new tab)</li>
-        <li><a href="https://www.facebook.com/sharer/sharer.php?u=nhs.uk/coronavirus-status-checker/" target="_blank">Share on Facebook</a> (opens in a new tab)</li>
-
-      </ul>
+    
+        <p><svg class="nhsuk-icon nhsuk-icon__twitter" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
```

Comment:
@davidhunter08 I added these as paragraph elements for now. I imagine a unordered list with `list-style-type: none;` would be better?